File: features/merchant/reputation/src/main/java/com/tokopedia/tkpd/tkpdreputation/inbox/domain/model/inboxdetail/ImageAttachmentDomain.java
package com.tokopedia.tkpd.tkpdreputation.inbox.domain.model.inboxdetail;

/**
 * @author by nisie on 8/23/17.
 */

public class ImageAttachmentDomain {

    private int attachmentId;
    private String description;
    private String uriThumbnail;
    private String uriLarge;

    public ImageAttachmentDomain(int attachmentId, String description,
                                 String uriThumbnail, String uriLarge) {
        this.attachmentId = attachmentId;
        this.description = description;
        this.uriThumbnail = uriThumbnail;
        this.uriLarge = uriLarge;
    }

    public int getAttachmentId() {
        return attachmentId;
    }

    public String getDescription() {
        return description;
    }

    public String getUriThumbnail() {
        return uriThumbnail;
    }

    public String getUriLarge() {
        return uriLarge;
    }
}

File: features/merchant/review_inbox/src/main/java/com/tokopedia/review/feature/inbox/buyerreview/domain/model/inboxdetail/ImageAttachmentDomain.kt
package com.tokopedia.review.feature.inbox.buyerreview.domain.model.inboxdetail

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

/**
 * @author by nisie on 8/23/17.
 */
class ImageAttachmentDomain constructor(
    @SerializedName("attachmentID") @Expose val attachmentId: String,
    @SerializedName("imageThumbnailUrl") @Expose val uriThumbnail: String,
    @SerializedName("imageUrl") @Expose val uriLarge: String
)
